crates/iota/tests/snapshots/shell_tests__new_tests__gitignore_exists.sh.snap

```diff
@@ -0,0 +1,22 @@
+---
+source: crates/iota/tests/shell_tests.rs
+description: tests/shell_tests/new_tests/gitignore_exists.sh
+---
+----- script -----
+# Copyright (c) Mysten Labs, Inc.
+# Modifications Copyright (c) 2025 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+# check that iota move new correctly updates existing .gitignore
+mkdir example
+echo "existing_ignore" >> example/.gitignore
+iota move new example
+cat example/.gitignore
+
+----- results -----
+success: true
+exit_code: 0
+----- stdout -----
+existing_ignore
+
```

Comment:
No `build/*`?

Reply:
apparently it's not created by the `iota move new` command for us. Might be something they have upstream that we haven't merged yet